### PR TITLE
Floating ip create changes

### PR DESCRIPTION
**Pull request by Jaganathancse**

### CODE REVIEW

**Comment by Jaganathancse** [comment]
recheck

File: devsetup/scripts/edpm-deploy-instance.sh [hunk 1]
```diff
@@ -59,10 +59,10 @@ openstack network agent list
 # Create an instance
 openstack server show test || {
     openstack server create --flavor m1.small --image cirros --nic net-id=private test --security-group basic --wait
-    openstack floating ip show 192.168.122.20 || openstack floating ip create public --floating-ip-address 192.168.122.20
-    openstack server add floating ip test 192.168.122.20
+    openstack floating ip show 192.168.122.200 || openstack floating ip create public --floating-ip-address 192.168.122.200
```
Comment (karelyatin) [inline devsetup/scripts/edpm-deploy-instance.sh:62]: I think we better do:-
floatip=$(openstack floating ip create public -f value -c floating_ip_address)
openstack server add floating ip test $floatip

File: devsetup/scripts/edpm-deploy-instance.sh [hunk 2]
```diff
@@ -59,10 +59,10 @@ openstack network agent list
...
+    openstack server add floating ip test 192.168.122.200
 }
 openstack server list
 
 # check connectivity via FIP
-ping -c4 192.168.122.20
```
Comment (karelyatin) [inline devsetup/scripts/edpm-deploy-instance.sh:68]: ping -c4 $floatip

File: devsetup/scripts/edpm-deploy-instance.sh (hunk 1 above)
Comment (Jaganathancse) [reply devsetup/scripts/edpm-deploy-instance.sh:62]: Ack

File: devsetup/scripts/edpm-deploy-instance.sh (hunk 2 above)
Comment (Jaganathancse) [reply devsetup/scripts/edpm-deploy-instance.sh:68]: ack

**Comment by Jaganathancse** [comment]
recheck

**Review by karelyatin — APPROVED** [review]
/lgtm

**Pull request merged**